src/main/config/secor.common.properties

```diff
@@ -294,4 +294,10 @@ secor.message.transformer.class=com.pinterest.secor.transformer.IdentityMessageT
 
 # Set below property to true if you want to have the md5hash appended to your s3 path.
 # This helps for better partitioning of the data on s3. Which gives better performance while reading and writing on s3
-secor.s3.prefix.md5hash=false
\ No newline at end of file
+secor.s3.prefix.md5hash=false
+
+# After the given date, secor will upload files to the supplied s3 alternative path
+secor.s3.alter_path.date=
```

Comment:
We don't use '_' in the parameter name, you can either call it 'alter.path.date' or 'alternative.path.date'